common/ellipses.js

```diff
@@ -67,7 +67,14 @@
                             }
                         };      
 
-                        resizeRow();
+                        scope.$watchCollection('rowValues', function (v) {
+                            $timeout(function() {
+                                timerCount = 0;
+                                containsOverflow = false, oldHeights = [];
```

Comment:
I don't see `oldHeights` being set anywhere in this file. Is it suppose to be set somewhere?